src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -70,151 +70,189 @@ td.tribe_message {
 	border-color: #e6db55;
 	-webkit-border-radius: 3px;
 	   -moz-border-radius: 3px;
-	        border-radius: 3px;
+			border-radius: 3px;
 	border-style: solid;
 	border-width: 1px;
 	padding: 6px 12px;
 }
 
 /* = Plugin Screen
 =============================================*/
-table.plugins .tribe-plugin-update-message {
-	background: #d54e21; /* taken from colour scheme in list-tables.css */
-	color: white;
-	display: inline-table;
-	margin: 6px 0;
-	padding: 10px 12px;
-}
-
-table.plugins .tribe-plugin-update-message h4 {
-	display: inline;
-	font-weight: bold;
-	margin-right: 8px;
-}
-
-table.plugins .tribe-plugin-update-message h4:after {
-	content: ' \00BB ';
-}
+table.plugins {
+	.tribe-plugin-update-message {
+		background: #D54E21; /* taken from colour scheme in list-tables.css */
+		color:      white;
+		display:    inline-table;
+		margin:     6px 0;
+		padding:    10px 12px;
+
+		h4 {
+			display:      inline;
+			font-weight:  bold;
+			margin-right: 8px;
 
+			&:after {
+				content: ' \00BB ';
+			}
+		}
 
-table.plugins .tribe-plugin-update-message a {
-	color: white;
-	text-decoration: underline;
+		a {
+			color:           white;
+			text-decoration: underline;
+		}
+	}
 }
 
 /* = Settings Screen
 =============================================*/
 .tribe-settings-form {
 	max-width: 1000px;
-}
-.tribe-settings-form fieldset {
-	clear: both;
-	display: inline-block;
-	padding: 10px 0;
-}
-.tribe-settings-form legend {
-	float: left;
-	font-weight: bold;
-	margin-right: 20px;
-	width: 220px;
-}
-.tribe-settings-form fieldset.tribe-field-license_key legend {
-	width: auto;
-}
-.tribe-settings-form .tribe-field-wrap {
-	float: left;
-	max-width: 500px;
 
-	*:first-child {
-		margin-top: 0;
+	fieldset {
+		clear: both;
+		display: inline-block;
+		padding: 10px 0;
+
+		&.tribe-field-license_key legend {
+			width: auto;
+		}
+	}
+
+	legend {
+		float: left;
+		font-weight: bold;
+		margin-right: 20px;
+		width: 220px;
+	}
+
+	.tribe-field-wrap {
+		float: left;
+		max-width: 500px;
+
+		*:first-child {
+			margin-top: 0;
+		}
+	}
+
+	.tribe-field-checkbox_bool p.tooltip {
```

Comment:
@cliffordp We probably need to remove this CSS block.

Reply:
See https://github.com/moderntribe/tribe-common/pull/1099